src/workflows-app/RunDetails.js

```diff
@@ -249,10 +278,22 @@ export const BaseRunDetails = (
       () =>
         div([
           div({ style: { padding: '1rem 2rem 2rem' } }, [header]),
-          div({ style: { display: 'flex', justifyContent: 'space-between', padding: '1rem 2rem 2rem' } }, [
+          div({ style: { display: 'flex', justifyContent: 'space-between', padding: '1rem 2rem 1rem' } }, [
             h(WorkflowInfoBox, { workflow }, []),
             h(TroubleshootingBox, { name, namespace, logUri: workflow.workflowLog, submissionId, workflowId, showLogModal }, []),
           ]),
+          div({ style: { fontSize: 16, padding: '0rem 2.5rem 1rem' } }, [
+            span({ style: { fontWeight: 'bold' } }, ['Approximate workflow cost: ']),
+            renderInProgressElement(workflow),
+            `$${taskCostTotal.toFixed(2)}`,
+            h(
+              TooltipTrigger,
+              {
+                content: 'This total is an estimate rounded up to the nearest cent and does not include subworkflow costs.',
```

Comment:
Is it rounded up to the nearest cent? It doesn't look like that's what `toFixed` does. I think it would be good to make this consistent with the other tooltip - if we do want to round everything UP to the next cent, say that in both places. 

Suggested edit (assuming no change to rounding): "Approximate cost is calculated based on the the list price of the VMs used and does not include disk cost, subworkflow cost, or any cloud account discounts."

In this one we DO want to mention subworkflow cost, since it's a sum that users might expect to include subworkflow tasks.